darshan-util/pydarshan/darshan/experimental/plots/heatmap_handling.py

```diff
@@ -125,7 +125,7 @@ def get_rd_wr_dfs(
     for op_key in ops:
         # mypy can't tell that these keys are in fact
         # in the ``SegDict``, so just ignore the type
-        seg_key = op_key + "_segments"  # type: ignore
+        seg_key = op_key + "_segments"
```

Comment:
@nawtrey had reported a requirement for the `type: ignore` here, but with bleeding edge versions of all dependencies and the `mypy.ini` proposed here, `mypy` considers the presence of the ignore a failure because it isn't needed, so I've removed it.